src/Core/Event.js

```diff
@@ -18,17 +17,17 @@ function handleEvent (event) {
 			return
 
 		if (typeof callback === 'function') {
-			value = callback.call(instance, event, props, state, context)
+			value = callback.call(owner, event, props, state, context)
 		} else if (typeof callback.handleEvent === 'function') {
-			if (instance !== callback && callback[SymbolComponent] === SymbolComponent)
-				host = getComponentElement(instance = callback)
+			if (owner !== callback && callback[SymbolComponent])
+				host = getComponentElement(owner = callback)
 
 			value = callback.handleEvent(event, props, state, context)
 		}
 
-		if (value && instance)
-			getLifecycleReturn(host, value)
-	} catch (err) {
-		invokeErrorBoundary(host, err, 'on'+type+':'+getDisplayName(callback.handleEvent || callback), SharedErrorPassive)
+		if (value && owner[SymbolComponent])
+			getLifecycleState(host, value)
+	} catch (e) {
```

Comment:
prefer `err` for all `Error` type args
(`e` is generally used for events)

Reply:
Your right, that's a better convention.